apps/lead/models.py

```diff
@@ -218,12 +218,22 @@ def get_for(cls, user):
         )
         return qs.annotate(
             no_of_entries=models.Count('entry', distinct=True),
-            assessment_id=models.F('assessment')
+            assessment_id=models.F('assessment'),
+            entries_verified_count=models.Count('entry', filter=models.Q(entry__verified=True))
         )
```

Comment:
Lets annotate `percentage_of_entries_verified` as well. And in the `get_percentage_of_entries_verified` simply do a `getattr()`.